src/button/component.jsx

```diff
@@ -892,17 +892,59 @@ export const Button : Component<ButtonOptions> = create({
             type:     'function',
             required: false,
             get value() : Function {
+                let initialHeight;
+                const logInlineGuestOutOfViewPortOnlyOnce = once((data) => {
+                    info(`inline_guest_outside_viewport`, data);
+                    immediateFlush();
+                });
+
                 return function onResizeHandler() {
                     const container = this.container;
-                    const parentContainer = this.container.parentElement;
+                    if (!initialHeight) {
+                        initialHeight = container.offsetHeight;
+                    }
 
-                    if (container && parentContainer && container.offsetHeight > parentContainer.offsetHeight) {
-                        info(`button_taller_than_parent`, {
-                            height:       container.offsetHeight,
-                            parentHeight: parentContainer.offsetHeight
-                        });
-                        flushLogs();
+                    const getScrollOffsetY = () => {
+                        if (window.pageYOffset) {
+                            return window.pageYOffset;
+                        }
+
+                        if (document.documentElement) {
+                            return document.documentElement.scrollTop;
+                        }
+
+                        return 0;
+                    };
+
+                    const checkIfInlineGuestInViewport = () => {
```

Comment:
Why do we care about the viewport? Surely the main concern is if the buttons are overflowing outside the parent element, no?

Reply:
Actually, I think they're 2 different problems. 

This is the log for https://www.onbuy.com use case, this is their checkout page with SPBs at the bottom of the page.

<img width="1680" alt="Checkout 2019-06-26 23-28-38" src="https://user-images.githubusercontent.com/3213579/60242383-6e89f100-986a-11e9-93fe-aa2a4b0ea838.png">

This is how it looks like **after** the buyers click on the credit card icons
<img width="1792" alt="Checkout 2019-06-26 23-31-41" src="https://user-images.githubusercontent.com/3213579/60242517-ae50d880-986a-11e9-8c4c-0f9ddffd1987.png">

The problem is if the buyers don't scroll, they may not know that there is the inline guest form, and that is the case we want to log.